packages/client/package.json

```diff
@@ -29,9 +29,9 @@
     "SQLServer",
     "MongoDB"
   ],
-  "main": "index.js",
+  "main": "default.js",
   "browser": "index-browser.js",
-  "types": "index.d.ts",
+  "types": "default.d.ts",
```

Comment:
This is an attempt to fix https://github.com/newrelic/node-newrelic/issues/1984. I don't think there was anything particularly wrong with `index` here, but rather a bug or limitation with `require-in-the-middle` (used by New Relic). 

There's an interplay between `main` being different from `exports` and that makes the `Hook` of `require-in-the-middle` not work when that is the case, although `require` did load the module correctly via the exports map and not `main`. 

In other words, `Hook` is supposed to trigger when `require('@prisma/client')` is executed, but it does not if `main` and `exports` point to different files, although in the end it does not matter. Maybe an edge case or a bug.

Reply:
also `require-in-the-middle` is used by opentelemetry js, datadog, elastic to name a few, it will affect them as well

Reply:
Hey, could you please give `5.10.0-integration-fix-client-exports-map.8` a try?

Reply:
(Nothing should have fundamentally changed)

Reply:
`5.10.0-integration-fix-client-exports-map.8` still works. I think as long as the main matches the exported default it's a sufficient enough fix.

Reply:
Thanks!